python/examples/handler.py

```diff
@@ -56,20 +47,15 @@ def echo(request, response, proxy):
     print "echo"
     yield tornado.gen.sleep(1)
     # stream args right back to request side
-    response.argstreams = [
-        InMemStream(request.endpoint),
-        request.argstreams[1],
-        request.argstreams[2]
-    ]
+    response.set_header(request.get_header_s())
+    response.set_body(request.get_body_s())
 
 
 @tornado.gen.coroutine
 def slow(request, response, proxy):
     yield tornado.gen.sleep(random.random())
-    response.argstreams = [
-        InMemStream(),
-        InMemStream(),
-        InMemStream("done")]
+    yield response.write_body("done")
+    response.flush()
```

Comment:
flush twice , I think it's okay. Basically nothing happened. But we can raise error too.